src/pages/signup-affiliates-details/_benefits.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex } from '../../components/containers'
+import { Localize, localize } from 'components/localization'
+import Join from 'images/svg/signup-affiliate-details/join.svg'
+import Earn from 'images/svg/signup-affiliate-details/earn.svg'
+import Products from 'images/svg/signup-affiliate-details/products.svg'
+import Support from 'images/svg/signup-affiliate-details/support.svg'
+import device from 'themes/device.js'
+import { Header, Text } from 'components/elements'
+
+const StyledFlex = styled(Flex)`
+    @media ${device.tablet} {
```

Comment:
Any reason, why we are hiding this for tablet design ?

Reply:
You can use `Desktop` / `Mobile` component from 'containers`. If you don't have that, you may need to pull the latest master.

Reply:
Added `Desktop` component as wrapper and removed `StyledFlex` style rules.